.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: Build gwt-site-webapp
+on:
+  push:
+    branches: ['main']
```

Comment:
This prevents forks (i.e. contributors) from running the workflow before making a pull request.